sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_validate.py

```diff
@@ -34,7 +34,8 @@ def _validate_derived_metric_info(metric_info: DerivedMetricInfo) -> bool:
 
 
 def _validate_document_filter_group_info(doc_filter_group: DocumentFilterConjunctionGroupInfo) -> bool:
-    _validate_telemetry_type(doc_filter_group.telemetry_type)
+    if not _validate_telemetry_type(doc_filter_group.telemetry_type):
```

Comment:
Marking this spot for past question: "What does it mean and what do we do if it fails validation?"

Reply:
Filter is not applied.